eng/mgmt/mgmtmetadata/storage_resource-manager.txt

```diff
@@ -4,11 +4,11 @@ Commencing code generation
 Generating CSharp code
 Executing AutoRest command
 cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp.output-folder=C:\code\srpsdk\sdk\storage\Microsoft.Azure.Management.Storage\src\Generated
-2020-02-25 09:14:44 UTC
+2020-03-26 11:27:51 UTC
 Azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      master
-Commit:      2df17398750388e7a8e51eeedaac8bfe099e612b
+Commit:      8802867c407dcec7f373e5a263996bf077031532
 AutoRest information
 Requested version: latest
-Bootstrapper version:    autorest@2.0.4413
+Bootstrapper version:    autorest@3.0.6187
```

Comment:
Hi, it is recommended not to use the latest autorest because it causes some issues in the generated code. Please update your generate.ps1 and use 2.0.4413 instead. Thanks

Reply:
This is fixed, I have regenerate with 2.0.4413.
But I don't change the generate.ps1 in this PR..
Would you please help to notify us when can we switch to latest, and I will use 2.0.4413 instead of latest before that.